stix2/environment.py

```diff
@@ -152,3 +152,17 @@ def add(self, *args, **kwargs):
     def parse(self, *args, **kwargs):
         return _parse(*args, **kwargs)
     parse.__doc__ = _parse.__doc__
+
+    def created_by(self, obj):
+        """Retrieve the Identity refered to by the object's `created_by_ref`.
+
+        Args:
+            obj: The STIX object whose `created_by_ref` property will be looked
+                up.
+
+        Returns:
+            The STIX object's creator.
+        """
+
+        creator_id = obj.get('created_by_ref', '')
```

Comment:
If there is no `created_by_ref`, we should probably `return None` rather than calling the line below.